tests/Doctrine/Tests/ORM/Functional/Ticket/DDC2632Test.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Tools\Export\Driver\YamlExporter;
+use Doctrine\ORM\Tools\Export\Driver\XmlExporter;
+use Doctrine\Tests\ORM\Functional\DatabaseDriverTestCase;
+/**
+ * @group DDC-2632
+ */
+class DDC2632Test extends DatabaseDriverTestCase
+{
+    protected function _getType() {
+        if ( ! class_exists('Symfony\Component\Yaml\Yaml', true)) {
+            $this->markTestSkipped('Please install Symfony YAML Component into the include path of your PHP installation.');
+        }
+        return 'yaml';
+    }
+    /**
+     * @var \Doctrine\DBAL\Schema\AbstractSchemaManager
+     */
+    protected $_sm = null;
+    public function setUp() {
+        //  $this->useModelSet('cms');
+        parent::setUp();
+        $this->_sm = $this->_em->getConnection()->getSchemaManager();
+    }
+    /**
+     * @group DDC-2632
+     */
+    public function testFKDefaultValueOptionExportYaml() {
+        $exporter = new YamlExporter();
+
+        if ( ! $this->_em->getConnection()->getDatabasePlatform()->supportsForeignKeyConstraints()) {
+            $this->markTestSkipped('Platform does not support foreign keys.');
+        }
+        $user = new \Doctrine\DBAL\Schema\Table("ddc2059_user");

```

Comment:
This and following lines are really setting up a schema in both tests - can you deduplicate them and move them to a private method?